python/GafferUI/PathListingWidget.py

```diff
@@ -961,6 +1039,13 @@ def __init__( self ) :
 
 		self.__currentEventModifiers = QtCore.Qt.NoModifier
 
+		self.__highlightedIndex = None
+
+	def setHighlightedIndex( self, index ) :
+
+		self.__highlightedIndex = index
```

Comment:
I think this is another spot when we can't use a bare QModelIndex, but must instead use QPersistentModelndex.

Reply:
Thanks, updated in 0a8d768b78ac41a6bc7c346b76e4990bb767724c.